book/website/collaboration/leadership/leadership-opportunities.md

```diff
@@ -1,7 +1,15 @@
 (cl-leadership-opportunities)=
 # Creating Leadership Opportunities
 
-Add contents from the Book Dash social discussions on Emergent leadership and infrastructure roles.
+While leadership opportunities can emerge, it can be helpful to create clear opportunities and invite individuals to lead.  
+
+This can be done through inviting people for individual contributions and facilitating a culture where people feel empowered and trusted to lead when opportunities arise.  
+
+Once a community or project team has grown to a certain size, a more formal structure can help to ensure leadership responsibility scale accordingly to keep everyone supported.
+Being clear about the roles people can take on and the commitment that is expected from them will allow people to manage expectations around skills, workload, and responsibilities.  
+
+One way to do this is to have people commit for a certain period and re-evaluate regularly if more or other community members have emerged and should be formally recognised for their work.
```

Comment:
Should you also not evaluate if the person that temporarily committed has also succeeded? I find the two parts of these sentences a bit disconnected.

Reply:
Yes, rephrased to make it hopefully (?) clearer